examples/research_town_demo.py

```diff
@@ -17,6 +17,9 @@ def run_sync_experiment(
         'Stefanie Jegelka',
         'Silvio Lattanzi',
         'Rex Ying',
+        'Tim Althoff',
+        'Christos Faloutsos',
+        'Julian McAuley',
```

Comment:
I have added three more agent names. Otherwise, there would be errors in finding reviewers with the default reviewer number, which is 3 in current setting.

Reply:
nice, thanks